frontend/src/components/pages/CampsList/CampsTable.tsx

```diff
@@ -77,7 +77,7 @@ const CampsTable = ({
     };
 
     getCamps();
-  }, [year]);
+  }, [year, setCampDrawerInfo]);
```

Comment:
linter: https://stackoverflow.com/questions/58866796/understanding-the-react-hooks-exhaustive-deps-lint-rule

Wondering if this was the error we got with the infinite camps loop? https://bobbyhadz.com/blog/react-hooks-exhaustive-deps
> cause an error because objects and arrays are compared by reference in JavaScript.